### DISCUSSION TITLE
How to find a Existing Issue in the kolibri-design-system

### DISCUSSION THREAD

**Discussion by sruthin21** [learningequality/kolibri-design-system#778] [Q&A] [answered]
I am Sruthin Teja ,I know React and vue js and I am New to Open source . I setup the project kolibri-design-system locally And I wanted
to contribute to the project . Unable to find the Issues which is not assigned . If you can share some begineer friendly Issues I can work on it

**Comment by AlexVelezLl** [comment, marked as answer]
Welcome @sruthin21!! Thank you for your interest in contributing to KDS! For beginner-friendly issues, you can look at the [unassigned good first issues](https://github.com/learningequality/kolibri-design-system/issues?q=is%3Aissue+is%3Aopen+no%3Aassignee+label%3A%22good+first+issue%22+) in KDS! If you start feeling confident with the project, you can also look at the [unasssigned help wanted issues](https://github.com/learningequality/kolibri-design-system/issues?q=is%3Aissue+is%3Aopen+no%3Aassignee+label%3A%22help+wanted%22), there are some pretty interesting issues there :hugs:.

You can also see the list of unassigned ["help wanted"](https://github.com/search?q=org%3Alearningequality+is%3Aopen+is%3Aissue+label%3A%22help+wanted%22+no%3Aassignee&type=issues) or ["good first issue"](https://github.com/search?q=org%3Alearningequality+is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22+no%3Aassignee&type=issues) issues across all our repositories.

Please let us know if you have any questions.
